crates/sui-json-rpc/Cargo.toml

```diff
@@ -12,6 +12,7 @@ chrono.workspace = true
 fastcrypto.workspace = true
 jsonrpsee.workspace = true
 hyper.workspace = true
+http-body = "0.4"
```

Comment:
Do we need this?

Reply:
Workspace has v1.0

Reply:
This is needed for the json-rpc library that we have that i was avoiding upgrading right now :)